algorithms/libfuzzer/options.cpp

```diff
@@ -256,6 +256,10 @@ auto postProcess(
     return false;
   }
 
+  if (global.executor != fuzzuf::cli::ExecutorKind::NATIVE) {
+    EXIT("Unsupported executor: `%s`", global.executor.c_str());
+  }
```

Comment:
:memo:
libfuzzerではNative以外をサポートしない。なので、いままで通りNativeLinuxExecutorを使い続ける。
GlobalOptionのExecutorのデフォルト値は Native なので（include/fuzzuf/cli/global_fuzzer_options.hpp参照）、
ユーザーは特にExecutorを意識せずに済む。

Reply:
(English translation) libfuzzer does not support anything other than Native. Therefore, we continue to use NativeLinuxExecutor as before.
Since the default value of GlobalOption's Executor is Native (see include/fuzzuf/cli/global_fuzzer_options.hpp)
The user is not particularly aware of the Executor.

Reply:
This is for fail-safe. If the CLI accepts unsupported combinations of the options, it causes unexpected behaviors from the users. It is better to deny unacceptable options rather than just ignore the options in the CLI UX aspects.